lib/requirefy.js

```diff
@@ -17,17 +17,29 @@ module.exports = function requirefy(code, options, resolvePath) {
     return header;
   }
 
-  function removeRequirejs(code, head, tail) {
+  function removeRequirejs(code, head, tail, moveUseStrict, useStricRange) {
     var top = code.substring(0, head.start - 1);
     var mid = code.substring(head.end, tail.start);
     var bot = code.substring(tail.end);
 
+    if (moveUseStrict) {
+      var beforeUseStrict = mid.substring(0, useStricRange[0]);
```

Comment:
Thanks.
BTW who would have thought that they each have a separate implementation:

- [`substring`](https://github.com/iojs/io.js/blob/master/deps/v8/src/string.js#L678)
- [`substr`](https://github.com/iojs/io.js/blob/master/deps/v8/src/string.js#L711)
- [`slice`](https://github.com/iojs/io.js/blob/master/deps/v8/src/string.js#L544)

They each call same C++ method in the end ` %_SubString` but have some minor differences.
Similar enough though I'd argue ;)